File: 2A_1/cpp/potega.cpp
/*
 * potega.cpp
 */


#include <iostream>
using namespace std;

int potega_it(int p, int w) {
	int wynik = 1;
	for (int i = 0; i < w; i++) {
		wynik = wynik * p;
	}
	return wynik;
}

int main(int argc, char **argv)
{
	int podstawa = 0;
	int wykladnik = 0;
	cout << "Podaj podstawę i wykladnik:" << endl;
	cin >> podstawa;
	cin >> wykladnik;
	cout << "Wynik: " << potega_it(podstawa, wykladnik);
	return 0;
}

File: 2A_1/python/potega.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  potega.py
#  

def potega_it(p, w):
    wynik = 1
    for i in range(w):
        wynik = wynik * p
    return wynik


def main(args):
    podstawa = int(input("Podaj podstawę: "))
    wykladnik = int(input("Podaj wykładnik: "))
    print("Wynik:", potega_it(podstawa, wykladnik))
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
